src/SFML/Audio/SoundFileReaderFlac.hpp

```diff
@@ -128,8 +128,8 @@ class SoundFileReaderFlac : public SoundFileReader
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    FLAC__StreamDecoder* m_decoder;    //!< FLAC decoder
-    ClientData           m_clientData; //!< Structure passed to the decoder callbacks
+    FLAC__StreamDecoder* m_decoder{nullptr}; //!< FLAC decoder
+    ClientData           m_clientData;       //!< Structure passed to the decoder callbacks
```

Comment:
Good catch. I took this further and made it so that default constructing `ClientData` intializes all members which also entailed making sure that `SoundFileReader::Info` can be default constructed.